x/amm/types/constants.go

```diff
@@ -6,6 +6,8 @@ import (
 
 const (
 	OneShareExponent = 18
+
+	BalancerGasFeeForSwap = 10_000
```

Comment:
Can you share what this does?

Reply:
here: https://github.com/elys-network/elys/blob/feat/swap-logic/x/amm/types/apply_swap.go#L7-L8

Reply:
I see - got it.